### PR TITLE
Handle area Damage or Heal

### PR DESCRIPTION
**Pull request by nagyjo**
- Create a new form template: setupAreaDamageForm.html
- add ShowAreaDamageForm function to display the new form
- add areaHealthModificationCtrl to handle the health change.
- add some css to design the new form.

### CODE REVIEW

File: static/app.js [hunk 1]
```diff
@@ -262,4 +278,40 @@
 		}
 	})
 
+	app.controller('areaHealthModificationCtrl', function($scope, $modalInstance, characters){
+		$scope.areaDamage = 0;
+
+		$scope.characters = characters;
+		$scope.characters.forEach(function (character) {
+			character.damaged = false;
```
Comment (initiativetracker) [inline static/app.js]: You shouldn't add damaged and damageHalved properties to the character prototype.

File: static/app.js [hunk 2]
```diff
@@ -262,4 +278,40 @@
...
+			character.damageHalved = false;
+		});
+
+		$scope.toggleDamaged = function (character) {
```
Comment (initiativetracker) [inline static/app.js]: If you tick damaged and damageHalved, and after you remove damaged flag, damageHalved remains ticked.

File: index.html [hunk 3]
```diff
@@ -67,6 +70,37 @@
...
 
+    <script type="text/ng-template" id="setupAreaDamageForm.html">
+      <form name="areaDamageForm" id="areaDamageForm">
+        Health change (negative if healed):
+        <input type="number" data-ng-model="areaDamage" onFocus="this.select();" autofocus>
+
+        <div class="scrollable medium-height">
+          <table id="charactersInAreaDamage" style="width: 100%">
+            <tr data-ng-repeat="character in characters">
+              <td>{{ character.uniqueName ? '' : '(#' + character.id + ')' }}{{ character.name }} ({{ character.initiation }})</td>
```
Comment (initiativetracker) [inline index.html]: Refactor this string to a function would be a good idea.
What do you think?

**Comment by nagyjo** [comment]
Fixed. Could yo check it?

File: index.html [hunk 4]
```diff
@@ -67,6 +70,37 @@
       </form>
     </script>
 
+    <script type="text/ng-template" id="setupAreaDamageForm.html">
+      <form name="areaDamageForm" id="areaDamageForm">
+        Health change (negative if healed):
+        <input type="number" data-ng-model="areaDamage" onFocus="this.select();" autofocus>
+
+        <div class="scrollable medium-height">
+          <table id="charactersInAreaDamage" style="width: 100%">
```
Comment (abrhm) [inline index.html]: Could you please move this style to the CSS?

Otherwise it looks good to go in!
Please squash the commits.

**Pull request merged**